docs/api/access-control.md

```diff
@@ -472,357 +441,3 @@ _NOTE:_ Even when returning `false`, this field _will_ be included in GraphQL
-
-<!-- TODO: Flesh this out more -->
-
-## Schema isolation
-
-By default, Keystone generates a single GraphQL endpoint with an
-include-by-default approach to what schema is available.
-
-In some cases (see [examples](#schema-isolation-examples) below), it is
-desirable to have an entirely separate schema available at different endpoints
-or to different users.
-
-<!-- TODO @jed: Insert links to discussions on security through obscurity & privacy of
-certain fields, etc -->
-
-<!-- TODO @jed: How does this fit in with the Admin UI? How can a new Admin UI
-be generated based on a given schema? -->
-
-### `createGraphQLMiddleware(config) => function(req, res, next)`
-
-Keystone exposes a `.createGraphQLMiddleware()` function which generates an
-express middleware you can then add to a route to serve your graphql api.
-
-`config` takes the form:
-
-```js
-var config = {
-  // Set the default value for all lists with a root level `access` option
-  // Takes the same form as the `access` option on lists
-  access: true,
-
-  // Configure individual lists
-  lists: {
-    // Lists are identified by the name passed to `.createList()`
-    [listKey]: {
-      // Set access config for this list.
-      // Takes the same form as the `access` option on lists
-      access: {
-        create: true,
-        read: true,
-        update: true,
-        delete: true,
-
-        // Overwite access config for specific fields
-        fields: {
-          // Overwrite the `.createList()` config to give full CRU access for
-          // admin
-          email: true,
-          password: true,
-        },
-      },
-    },
-  },
-};
-```
-
-### Schema Isolation Examples
-
-#### Role based schemas
-
-```js
-keystone.createList('User', {
-  fields: {
-    name: { type: Text },
-    address: { type: Text },
-    email: {
-      type: Email,
-      access: ({ existingItem, auth }) => existingItem.id === auth.id,
-    },
-    password: {
-      type: Password,
-      access: {
-        read: false,
-        update: ({ existingItem, auth }) => existingItem.id === auth.id,
-      },
-    },
-  },
-});
-
-// This schema assumes if you can hit the route, then you have god-mode.
-const adminSchema = keystone.createGraphQLMiddleware({
-  // grant access to everything by default, however is not enough to overwrite
-  // the specific rules setup in the above `.createList()`
-  access: true,
-  lists: {
-    // Overrides for specific fields
-    User: {
-      access: {
-        fields: {
-          // Overwrite the `.createList()` config to give full CRUD access for
-          // admin
-          email: true,
-          password: true,
-        },
-      },
-    },
-  },
-});
-
-const publicSchema = keystone.createGraphQLMiddleware();
-
-app.use('/graphql', (req, res, next) => {
-  // Assuming req.user is set by some earlier authentication middleware
-  if (req.user.role === 'ADMIN') {
-    return adminSchema(req, res, next);
-  }
-  publicSchema(req, res, next);
-});
-```
```

Comment:
Happy to remove this 👍 

Did you capture it somewhere else?

Reply:
Yep in notion